lib/metrics/index.go

```diff
@@ -0,0 +1,134 @@
+/*
+Copyright 2024 Huawei Cloud Computing Technologies Co., Ltd.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metrics
+
+var ComIndexRegistry = map[string]*Metric{
+	"httpd": {
+		HelpMap: map[string]string{
+			"fieldsWritten":       "The number of field columns written",
+			"writeReqBytes":       "Total amount of data written successfully (as Byte)",
+			"writeReqBytesIn":     "The total amount of write data (including write failures and write successes)",
+			"writeReq":            "The cumulative number of write requests received by the serverThe cumulative number of write requests received by the server",
+			"writeReqActive":      "The number of currently active write requests",
+			"writeReqDurationNs":  "Cumulative write data delay (NS)",
+			"pointsWrittenOK":     "Number of successful bars written",
```

Comment:
It's not a good name. pointsWrittenSuccess might be better, `OK` is more oral?

Reply:
These are the names of the kernel monitoring metrics collected by openGemini. They already exist in the database and have not been newly added by this function.
To standardize the names of these metrics, we need to modify the code of the kernel collection metrics and the matching Grafana monitoring panel. This issue is not related to the functions developed this time. But we may submit another PR for modification.